sdk/src/main/java/com/antithesis/sdk/Random.java

```diff
@@ -46,16 +46,19 @@ public static long getRandom() {
      * @param <T>   Type of the array member items
      * @return Randomly selected item from the provided array.
      */
-    public static <T> Optional<T> randomChoice(T[] array) {
+    //public static <T> Optional<T> randomChoice(T[] array) {
+    public static <T> T randomChoice(T[] array) {
```

Comment:
I'm inclined to believe we want this to be List<T> to be in line with the Java collections behavior. 

As it stands, it is hard to do something like randomChoice(Integer[] arr) because it is unintuitive to create an Integer[ ] relative to an int[]

Reply:
Resolved in next commit